styles/namemc/catppuccin.user.css

```
@@ -52,12 +52,59 @@
     @crust: @catppuccin[@@lookup][@crust];
     @accent-color: @catppuccin[@@lookup][@@accent];
 
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    #bsButton(@col, @bgCol) {
+      --bs-btn-color: @col;
+      --bs-btn-hover-color: darken(@col, 5%);
+      --bs-btn-active-color: darken(@col, 5%);
+      --bs-btn-disabled-color: @col;
+
+      --bs-btn-bg: @bgCol;
+      --bs-btn-hover-bg: darken(@bgCol, 5%);
+      --bs-btn-active-bg: darken(@bgCol, 5%);
+      --bs-btn-disabled-bg: @bgCol;
+      --bs-btn-border-color: @bgCol;
+      --bs-btn-hover-border-color: @bgCol;
+      --bs-btn-active-border-color: @bgCol;
+      --bs-btn-disabled-border-color: @bgCol;
+    }
+    #bsButtonOutline(@col, @hover) {
+      --bs-btn-color: @col;
+      --bs-btn-hover-bg: @col;
+      --bs-btn-active-bg: @col;
+      --bs-btn-border-color: @col;
+      --bs-btn-disabled-color: @col;
+      --bs-btn-focus-shadow-rgb: @col;
+      --bs-btn-hover-border-color: @col;
+      --bs-btn-active-border-color: @col;
+      --bs-btn-disabled-border-color: @col;
+
+      --bs-btn-hover-color: @hover;
+      --bs-btn-active-color: @hover;
+    }
+
+    #rgbify(@color) {
+      @rgb-raw: red(@color), green(@color), blue(@color);
+    }
+
     --bs-body-color: @text;
-    --bs-body-color-rgb: @text;
+    --bs-body-color-rgb: #rgbify(@text);
```

You need to unfortunately add these brackets after every #rgbify() usage:
```suggestion
    --bs-body-color-rgb: #rgbify(@text) [];
```